content/integrations/faq/how-can-i-gather-metrics-from-the-unix-shell.md

```diff
@@ -1,23 +1,49 @@
 ---
 title: How can I gather metrics from the UNIX shell?
 kind: faq
+disable_toc: true
+further_reading:
+- link: "developers/"
+  tag: "Documentation"
+  text: "Developer Tools"
+- link: "developers/integrations/new_check_howto/"
+  tag: "Documentation"
+  text: "Create a new Integration"
 ---
 
-To gather metrics from the UNIX command line, we can use the unannounced shell integration.
+To gather metrics from the UNIX command line, try the shell integration.
 
-This integration has not yet been merged into the master branch, install this check as a custom Agent check in the appropriate Datadog Agent [directories][1].
+<div class="alert alert-warning">
+Note: This integration is experimental and not actively developed. 
+</div>
 
-You can find the integration files on GitHub:
+## Setup
 
-* [Shell YAML Example][2]
-* [Shell checks.d][3]
+### Installation
 
-This solution is a good alternative to creating a custom Agent check for data you can easily gather directly from the UNIX shell. For example, sending a metric with a value of the number of files in a certain directory.
+Install the shell integration as a [custom Agent check][1]. The integration files are on GitHub:
 
-Caveat: 
+| File        | Agent Directory |
+|-------------|-----------------|
+| [YAML][2]   | `conf.d`        |
+| [Python][3] | `checks.d`      |
+
+**Note**: The names of the YAML and Python files must match.
+
+### Configuration
+
+Update the [shell YAML file][2] as necessary.
 
 The user that the Agent runs as may need sudo access for the shell command. Sudo access is not required when running the Agent as root (not recommended).
 
-[1]: /agent/agent_checks/#directory
-[2]: https://github.com/DataDog/dd-agent/blob/garner/shell-integration/conf.d/shell.yaml.example
-[3]: https://github.com/DataDog/dd-agent/blob/garner/shell-integration/checks.d/shell.py
+## Use Case
+
+The shell integration is an alternative to creating a custom Agent check. It collects data directly from the UNIX shell, for example: sending a metric with a value of the number of files in a certain directory.
```

Comment:
Actually it's the very same thing, `shell.py` from the linked repo is indeed a custom check. Do we really need this page provided that we already have https://docs-staging.datadoghq.com/ruth/unix-shell-metrics/developers/write_agent_check/?tab=agentv6#writing-checks-that-run-command-line-programs?

Reply:
@masci - If you feel the page is unnecessary, we can remove it and point the URL to writing checks that run command line programs.